docs/site/src/content/docs/plugins/language/complexity-check.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Complexity Check Plugin
+---
```

Comment:
Are these pages supposed to be hidden from the docs site? I pulled down the branch and ran `just start-docs` and I can't find these pages. Maybe I'm doing something wrong?